src/odemis/driver/technolution.py

```diff
@@ -31,6 +30,7 @@
 import re
 import threading
 import time
+from distutils.version import StrictVersion
```

Comment:
So far we didn't have a dependency on distutils. Could we avoid it? It just one tiny function. You could either use `pkg_resources.parse_version()`, as in gui.util.updater, or do as in xt_client, and just "open code it" as ` [int(u) for u in s.split(".")]`.

Reply:
We could avoid it, but distutils is in the standard library, what is the disadvantage of adding dependencies from the standard library?

Reply:
Oh... then in general it would be completely fine. However, I just saw in the doc that `distutils` is deprecated and will be removed in Python 3.12! So let's not depend on it now :confused:

Reply:
Then I will replace it